### DISCUSSION TITLE
get impossible (https install)

### DISCUSSION THREAD

**Discussion by alecristia** [LAAC-LSCP/ChildProject#252] [Q&A] [answered]
I'm trying to get converted annotations from the solomon corpus.

I did: 

$ datalad get annotations/vtc/converted/
> [ERROR  ] not available; (Note that these git remotes have annex-ignore set: origin) [get(/Users/acristia/Documents/gitrepos/solis_metrics/input/solomon/annotations/vtc/converted/01_CW01_CH01_FB03_FB11_190622_0_0.csv)] 
...
[ERROR  ] not available; (Note that these git remotes have annex-ignore set: origin) [get(/Users/acristia/Documents/gitrepos/solis_metrics/input/solomon/annotations/vtc/converted/01_CW01_CH01_FB10_FB18_190729_0_0.csv)] 
...
[WARNING] could not get some content in /Users/acristia/Documents/gitrepos/solis_metrics/input/solomon/annotations/vtc/converted ['/Users/acristia/Documents/gitrepos/solis_metrics/input/solomon/annotations/vtc/converted/01_CW01_CH01_FB03_FB11_190622_0_0.csv'
...
action summary:
  get (error: 383, impossible: 1)

I checked my connection, and it seems fine.

Might this be due to the fact that I mistakenly installed using https, rather than the recommended ssh?

**Comment by alecristia** [comment, marked as answer]
The answer is yes:

I removed the dataset (with `datalad remove solomon`)
then reinstalled, and get now works.

**Comment by lucasgautheron** [comment]
Hello,

The dataset has been moved to GIN (https://gin.g-node.org/LAAC-LSCP/solomon)
I recommend using this version from now on;


Le jeu. 8 juil. 2021 à 18:50, Alex Cristia ***@***.***> a
écrit :

> The answer is yes:
>
> I removed the dataset (with datalad remove solomon)
> then reinstalled, and get now works.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/LAAC-LSCP/ChildProject/discussions/252#discussioncomment-980540>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABI4QHXCPGESCL6B3TUA3YTTWXJMPANCNFSM5ABDW75Q>
> .
>


-- 
Lucas Gautheron
06 79 23 86 47

Laboratoire de Sciences Cognitives et Psycholinguistique
Bureau 414 - 29 rue d'Ulm
75005 Paris
